### PR TITLE
[REVIEW] fix incorrect slot-size-array in the HugeCTR training nb

### PR DESCRIPTION
**Pull request by rnyak**
This is to address this bug. https://github.com/NVIDIA/NVTabular/issues/837

### CODE REVIEW

**Comment by rnyak** [comment]
@vinhngx Thanks again for reporting this. Can you please take a look at this PR? thanks.

**Comment by rnyak** [comment]
@benfred I updated the inference notbook tag as `0.5.3` in the readme, bcs that's what it worked for me. fyi.

File: examples/getting-started-movielens/inference-HugeCTR/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Inference with HugeCTR Model
+# Training and Inference with HugeCTR Model
 
-In this folder, we provide two example notebooks, [movielens-HugeCTR](https://github.com/NVIDIA/NVTabular/tree/main/examples/inference_triton/inference-HugeCTR/movielens-HugeCR.ipynb) and [movielens-HugeCTR-inference](https://github.com/NVIDIA/NVTabular/tree/main/examples/inference_triton/inference-HugeCTR/movielens-inference.ipynb), and explain the steps to do inference with Merlin Inference API after training a model with HugeCTR framework. 
+In this folder, we provide two example notebooks, [Training-with-HugeCTR](https://github.com/rnyak/NVTabular/blob/fix-hugectr-nbs/examples/getting-started-movielens/inference-HugeCTR/Training-with-HugeCTR.ipynb) and [Triton-Inference-with-HugeCTR](https://github.com/rnyak/NVTabular/blob/fix-hugectr-nbs/examples/getting-started-movielens/inference-HugeCTR/Triton-Inference-with-HugeCTR.ipynb), and explain the steps to do inference with Merlin Inference API after training a model with HugeCTR framework.
```
Comment (benfred) [inline examples/getting-started-movielens/inference-HugeCTR/README.md:3]: We shouldn't have the links going to your fork here -

Comment (rnyak) [reply examples/getting-started-movielens/inference-HugeCTR/README.md:3]: oh definitely, good catch, will fix it asap :)

**Review by benfred — APPROVED** [review]

**Pull request merged**